recipes/pyqstem/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pyqstem" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 54dc45800334e886cd4e2bf4d8f8139b7f904c367895a350303767d1eca7eaa4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cython
+    - fftw
+    - numpy
+    - python
+    - setuptools
```

Comment:
numpy, python, fftw, and setuptools should be in `host`, not `build`

Reply:
Thanks, I will try that - hopefully it will make it build correctly :-)